ln-gateway/src/lib.rs

```diff
@@ -27,6 +27,8 @@ use tokio::sync::{mpsc, oneshot};
 use tracing::{debug, error, instrument, warn};
 use webserver::run_webserver;
 
+pub type LnGatewayResult<T> = std::result::Result<T, LnGatewayError>;
```

Comment:
Let's just call it `Result`. We do that elsewhere and I think it's understandable.